Tests/TokamakCLITests/TokamakLintTests.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  SwiftSyntaxTests.swift
+//  TokamakCLI
+//
+//  Created by Matvii Hodovaniuk on 3/31/19.
+//
+
+import TokamakLint
+import XCTest
+
+let srcRoot = ProcessInfo.processInfo.environment["SRCROOT"]!
+
+final class SwiftSyntaxTests: XCTestCase {
+  func testAllFiles() throws {
+    let linter = TokamakLint()
+    let path = "\(srcRoot)/"
+    try linter.lintFolder(path)
+  }
+
+  func testFile() throws {
+    let linter = TokamakLint()
+    let path = "\(srcRoot)/Sources/Tokamak/Components/Host/Alert.swift"
+    try linter.lintFile(path)
+  }
+
+  func testPropsIsEquatable() throws {
+    let linter = TokamakLint()
+    let path = "\(srcRoot)/Tests/TokamakCLITests/TestPropsEquatable.swift"
+    XCTAssertTrue(try linter.isPropsEquatable(path))
```

Comment:
we should provide a negative test case here as well with a separate file, which does not have `Props` type conforming to `Equatable` and it would be verified that `isPropsEquatable` returns `false` for that file.